dev/tests/integration/framework/Magento/TestFramework/Helper/Curl.php

```diff
@@ -1,30 +0,0 @@
-<?php
```

Comment:
It is not recommended to delete the existing file, as other third-party extensions might use it. Instead, mark it as `deprecated`.

Thanks